model/src/components/types.ts

```diff
@@ -134,6 +134,22 @@ export interface TelephoneNumberFieldComponent extends TextFieldBase {
   }
 }
 
+export interface FileUploadFieldComponent {
+  type: ComponentType.FileUploadField
+  name: string
+  title: string
+  hint?: string
+  options: {
+    required?: boolean
+    accept?: string
+  }
+  schema: {
+    max?: number
+    min?: number
+    length?: number
```

Comment:
Should this be `contentLength` like in the **forms-runner** PR?

Do any others need to match?

https://github.com/DEFRA/forms-runner/pull/319/files#diff-af9783aacc879f766e29d9da32da6a569cef4a809581025ecf8fe69dd6cb7e5bR21-R25

Reply:
I'll revisit this after with Dave